### DISCUSSION TITLE
Can you modify the onboarding PR?

### DISCUSSION THREAD

**Discussion by kevin-matthews-vc** [renovatebot/renovate#27613] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I want to modify the onboarding PR, i know this can be done self hosted, but via the mend version I can't find anything within the UI that would allow me to specify what i want the defaults of the renovate/configure branch to be, meaning i have to go in to every one to modify it and add the base config i want.

Am i missing something?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by kevin-matthews-vc** [comment]
looking a bit closer through this it is looking for a repo by default called 
`renovate-config` 

However in the documentation although it does suggest this repo name it says you can call this whatever you want. However this isnt the case if you want the renovate-config to be picked up automatically.

**Reply by rarkins** [reply, marked as answer]
If you want it picked up automatically, it needs to follow a naming convention. Otherwise Renovate does not have enough API credits to try every filename possible until it finds yours

**Reply by kevin-matthews-vc** [reply]
Thinking more of a few settings in the ui , to specify things like base config repo - basically env variable overrides
